templates/hexagonal-spring-boot/jpa-adapter/src/main/java/packageName/repository/config/JpaAdapterConfig.java

```diff
@@ -4,17 +4,17 @@
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
-import packageName.domain.port.ExampleObtainPoem;
-import packageName.repository.ExamplePoetryRepository;
-import packageName.repository.dao.ExamplePoetryDao;
+import packageName.domain.port.ObtainExample;
+import packageName.repository.ExampleRepository;
+import packageName.repository.dao.ExampleDao;
 
 @Configuration
```

Comment:
No need of **@Configuration** annotation here.

Reply:
I need to let know that this is a configuration more like a marker too for the spring to understand that it is a configuration

Reply:
But @Configuration is used to load the configuration while spring boots. We are already importing this file in BootstrapConfig, aren't we?

Reply:
It is to loose couple it. Now maybe we have BootstrapConfig future can be something else too

Reply:
Ok I have faced problem with this kind of setup in past, when I have 2 JpaAdapters for a single port. I get errors telling like there are 2 beans of same type

Reply:
> Ok I have faced problem with this kind of setup in past, when I have 2 JpaAdapters for a single port. I get errors telling like there are 2 beans of same type

Shouldn't then we have unique config names ?

Reply:
I'll approve this and probably come back to you with an example for the issue I faced earlier. May be you are right but I have to give it a try.